backend/src/main/java/com/woowacourse/moamoa/study/service/MyStudyService.java

```diff
@@ -51,4 +60,24 @@ private List<MyStudyResponse> mapToResponse(final List<MyStudySummaryData> myStu
                 ))
                 .collect(Collectors.toList());
     }
+
+    public MyRoleResponse findMyRoleInStudy(final Long githubId, final Long studyId) {
+        final Member member = memberRepository.findByGithubId(githubId)
+                .orElseThrow(MemberNotFoundException::new);
+        final Study study = studyRepository.findById(studyId)
+                .orElseThrow(StudyNotFoundException::new);
+
+        final Participants participants = study.getParticipants();
+        return getMyRoleResponse(member, participants);
+    }
+
+    private MyRoleResponse getMyRoleResponse(final Member member, final Participants participants) {
+        if (Objects.equals(participants.getOwnerId(), member.getId())) {
+            return new MyRoleResponse(MemberRole.OWNER);
+        }
+        if (participants.isParticipate(member.getId())) {
+            return new MyRoleResponse(MemberRole.MEMBER);
+        }
+        return new MyRoleResponse(MemberRole.NON_MEMBER);
+    }
```

Comment:
```suggestion
    Role role = study.getRole(member.getId());
    return new MyRoleResponse(role);
```
이런 식으로 메시지를 날리면 좋을 것 같아요

Reply:
그리고 Domain에 단위 테스트를 만들면 좋을 것 같아요.

Reply:
로직 이동하고 단위 테스트 생성했습니다 ~